internal/cli/atlas/serverless/backup/snapshots/describe.go

```diff
@@ -26,8 +26,8 @@ import (
 	"github.com/spf13/cobra"
 )
 
-const describeTemplate = `ID	SNAPSHOT TYPE	TYPE	DESCRIPTION	EXPIRES AT
-{{.ID}}	{{.SnapshotType}}	{{.Type}}	{{.Description}}	{{.ExpiresAt}}
```

Comment:
We do not receive Type and Description anymore in the new object.
```
v1:
type CloudProviderSnapshot struct {
    Id                     string    `json:"id,omitempty"`
    RetentionInDays        int       `json:"retentionInDays,omitempty"`
    CreatedAt              string    `json:"createdAt,omitempty"`
    ExpiresAt              string    `json:"expiresAt,omitempty"`
    Description            string    `json:"description,omitempty"`
    Links                  []*Link   `json:"links,omitempty"`
    MasterKeyUUID          string    `json:"masterKeyUUID,omitempty"`
    MongodVersion          string    `json:"mongodVersion,omitempty"`
    SnapshotType           string    `json:"snapshotType,omitempty"`
    Status                 string    `json:"status,omitempty"`
    StorageSizeBytes       int       `json:"storageSizeBytes,omitempty"`
    Type                   string    `json:"type,omitempty"`
    CloudProvider          string    `json:"cloudProvider,omitempty"`
    Members                []*Member `json:"members,omitempty"`
    ReplicaSetName         string    `json:"replicaSetName,omitempty"`
    SnapshotsIds           []string  `json:"snapshotIds,omitempty"`
    ServerlessInstanceName string    `json:"serverlessInstanceName,omitempty"`
}

v2:
type ServerlessBackupSnapshot struct {
    CreatedAt              *time.Time `json:"createdAt,omitempty"`
    ExpiresAt              *time.Time `json:"expiresAt,omitempty"`
    FrequencyType          *string    `json:"frequencyType,omitempty"`
    Id                     *string    `json:"id,omitempty"`
    Links                  []Link     `json:"links,omitempty"`
    MongodVersion          *string    `json:"mongodVersion,omitempty"`
    ServerlessInstanceName *string    `json:"serverlessInstanceName,omitempty"`
    SnapshotType           *string    `json:"snapshotType,omitempty"`
    Status                 *string    `json:"status,omitempty"`
    StorageSizeBytes       *int64     `json:"storageSizeBytes,omitempty"`
}
```

I verified v1 command and the two fields are not populated. Also as per the [docs](https://www.mongodb.com/docs/atlas/reference/api/cloud-backup/backup/return-one-snapshot-of-one-serverless-instance/#response), there is no mention of type and description and hence removed them from the template.

Reply:
SGTM